src/server/pps/cmds/cmds.go

```diff
@@ -235,6 +237,8 @@ $ pachctl list-job -p foo bar/YYY
 		}),
 	}
 	rawFlag(listDatum)
+	listDatum.Flags().BoolVar(&paginate, "paginate", false, "Get a subset of the results")
+	listDatum.Flags().Int64Var(&page, "page", 0, "Specify the page of results to send")
```

Comment:
Ok, after offline disc going w int64's ... but also we're using `pageSize` (a new param) w a non zero value as the measure if we should paginate.